app/controllers/new_admin/allocation_chart_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module NewAdmin
+  class AllocationChartController < ApplicationController
+    layout "new_admin"
+
+    def index
+      allocations = AllocationsAndUnalocatedUsersQuery.new(Allocation).call
+
+      @allocations = allocations.map { |allocation| decorated_allocation(allocation) }
```

Comment:
you can just call allocations.decorate

Reply:
you mean like this? 
```rb
@allocations = allocations.decorate
```

it doesn't seem to work

Reply:
The right syntax is to use the collection decorate. @samkevin1, pay attention that you're using the draper to decorate the model. If you do not know about it, here is the documentation :) :

https://github.com/drapergem/draper#collections

Reply:
Yeah I wasn't understanding how to use it very well, but I changed it